src/sbt-test/sbt-plugins/simple/build.sbt

```diff
@@ -16,10 +16,18 @@ lazy val core = project.in(file("core")).settings(
 
 lazy val cli = project.in(file("cli")).dependsOn(core).enablePlugins(CliPlugin)
 
+lazy val stubbedDeployNpmBuild = taskKey[Unit]("Verify stubbing the deploy npm build works")
+
 lazy val webService = project.in(file("webservice")).dependsOn(core).enablePlugins(WebServicePlugin)
 
 lazy val webApp = project.in(file("webapp")).dependsOn(core).enablePlugins(WebappPlugin)
-  .settings(NodeKeys.nodeProjectDir in Npm := file("client"))
+  .settings(
```

Comment:
This is a dummy task just for the purpose of testing. The custom task key is defined above in this file. I can walk you through how scripted testing works if you like.